seqr/views/apis/staff_api_tests.py

```diff
@@ -375,6 +375,45 @@
+    "subject_id": "NA20878"
+}
+
+SAMPLE_QC_DATA = [
+    's	PROJECT	DATA_TYPE	RESEARCH_PROJECT_NUMBER	PRODUCT	INDIVIDUAL_NAME	ANALYSIS_END	PCT_CONTAMINATION	AL_PCT_CHIMERAS	WGS_MEAN_COVERAGE	WGS_MEDIAN_COVERAGE	HS_MEAN_TARGET_COVERAGE	HS_PCT_TARGET_BASES_20X	ALIGNER	MYRANK	seqr_id	filtered_callrate	data_type	filter_flags	qc_platform	known_pop	pca_scores	qc_pop	prob_afr	prob_amr	prob_asj	prob_eas	prob_fin	prob_mde	prob_nfe	prob_sas	pop_PC1	pop_PC2	pop_PC3	pop_PC4	pop_PC5	pop_PC6	fail_n_snp	fail_r_ti_tv	fail_r_insertion_deletion	fail_n_insertion	fail_n_deletion	fail_r_het_hom_var	qc_metrics_filters	sample_qc.call_rate	sample_qc.n_called	sample_qc.n_not_called	sample_qc.n_filtered	sample_qc.n_hom_ref	sample_qc.n_het	sample_qc.n_hom_var	sample_qc.n_non_ref	sample_qc.n_singleton	sample_qc.n_snp	sample_qc.n_insertion	sample_qc.n_deletion	sample_qc.n_transition	sample_qc.n_transversion	sample_qc.n_star	sample_qc.r_ti_tv	sample_qc.r_het_hom_var	sample_qc.r_insertion_deletion	sample_qc.f_inbreeding.f_stat	sample_qc.f_inbreeding.n_called	sample_qc.f_inbreeding.expected_homs	sample_qc.f_inbreeding.observed_homs\n',
+    '0002_001-1A_2	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	WES	[]	Unknown	Unknown	[0.05828154658288453,0.055632660318004644,-0.013526954385677643,-0.00672922733297315,-0.02231288278252597,-0.01851126825854067]	nfe	0.0000e+00	0.0000e+00	0.0000e+00	0.0000e+00	0.0000e+00	0.0000e+00	1.0000e+00	0.0000e+00	5.8282e-02	5.5633e-02	-1.3527e-02	-6.7292e-03	-2.2313e-02	-1.8511e-02	true	false	false	true	true	false	["n_deletion","n_insertion","n_snp"]	7.4665e-01	15368781	5214800	0	15179462	122960	66359	189319	638	211270	20345	24063	144573	66697	0	2.1676e+00	1.8530e+00	8.4549e-01	5.3433e-01	15368781	1.5105e+07	15245821\n',
+    '0004_001-1A	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	WES	[]	Unknown	Unknown	[0.057975465404746004,0.05783022522596767,-0.007880071185891558,-0.005536870286142893,-0.03154735973764001,-0.010849047729001934]	nfe	1.0000e-02	0.0000e+00	0.0000e+00	0.0000e+00	0.0000e+00	0.0000e+00	9.9000e-01	0.0000e+00	5.7975e-02	5.7830e-02	-7.8801e-03	-5.5369e-03	-3.1547e-02	-1.0849e-02	true	false	false	true	true	false	["n_deletion","n_insertion","n_snp"]	7.5422e-01	15524580	5059001	0	15331598	124440	68542	192982	627	216123	20773	24628	147636	68487	0	2.1557e+00	1.8155e+00	8.4347e-01	5.3262e-01	15524580	1.5258e+07	15400140\n',
+    '0006_001-1A_2	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	NA	WES	[]	Unknown	Unknown	[0.05456973953775274,0.05595057043008969,-0.008577987268070675,0.010774467092638367,-0.032619925661112215,0.010606945679118955]	nfe	0.0000e+00	0.0000e+00	0.0000e+00	0.0000e+00	0.0000e+00	0.0000e+00	1.0000e+00	0.0000e+00	5.4570e-02	5.5951e-02	-8.5780e-03	1.0774e-02	-3.2620e-02	1.0607e-02	true	false	false	true	true	false	["n_deletion","n_insertion","n_snp"]	7.4155e-01	15263762	5319819	0	15076982	120766	66014	186780	536	209877	19336	23581	143515	66362	0	2.1626e+00	1.8294e+00	8.1998e-01	5.2956e-01	15263762	1.5007e+07	15142996\n',
```

Comment:
Only include the relevant columns here

Reply:
Sorry, I didn't understand the 'qc' data. Most of the un-relevant columns are removed.